.github/workflows/superset-python-integrationtest.yml

```
@@ -67,6 +68,9 @@ jobs:
           ./scripts/python_tests.sh
       - name: Upload code coverage
         if: steps.check.outputs.python
+        continue-on-error: true # codecob is not critical and randomly fails at times
```

```suggestion
        continue-on-error: true # codecov is not critical and randomly fails at times
```
Code on the Cob?! Sign me up!

This still upholds/blocks on the standards (e.g. core packages at 100% coverage) right?